app/routes/surveys/components/SurveyPage.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+import React, { Component } from 'react';
+import SurveyList from './SurveyList';
+import styles from './surveys.css';
+import type { SurveyEntity } from 'app/reducers/surveys';
+import { ListNavigation } from '../utils.js';
+import TextInput from 'app/components/Form/TextInput';
+import Content from 'app/components/Layout/Content';
+
+type Props = {
+  surveys: Array<SurveyEntity>,
+  fetching: boolean,
+  addSurvey: SurveyEntity => Promise<*>,
+  deleteSurvey: number => Promise<*>,
+  push: string => void
+};
+
+type State = {
+  searchQuery: string
+};
+
+export default class SurveyPage extends Component<Props, State> {
+  state = {
+    filters: {},
+    searchQuery: ''
+  };
+
+  surveySearch = (surveys: Array<Object>) =>
+    surveys.filter(
```

Comment:
Maybe a bit out of the scope of this PR, but im not a fan of such "inline" searching algorithms. Searching should be done generic with fuzz (fuzzysearching) through all data fields, or just simply via elasticsearch.

Reply:
Fair feedback, I don't disagree. Would be nice to get this PR in so I could start working on the remaining tasks though, we could replace the search in another pr.

Reply:
Ofc. It is just a general thing.

Otherwise this looks good :smile: